getting-started/build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+    id 'io.quarkus.gradle.plugin' version '0.12.0'
+}
+/*
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.quarkus:quarkus-gradle-plugin:0.12.0'
+    }
+}
+*/
+repositories {
+     mavenCentral()
+     maven { url 'http://repository.jboss.org/nexus/content/groups/public' }
+}
+
+dependencies {
+    compileOnly group: 'io.quarkus', name: 'quarkus-resteasy', version:'0.12.0'
```

Comment:
We've changed this to from `compileOnly`to `implementation` in the docs. Also the version.

Reply:
Sure, done.